server/views/pages/bookVisit/checkVisitDetails.njk

```diff
@@ -8,3 +9,14 @@
   {{ bookingJourney | dump(2) }}
   
 </pre>
+
+<form action="/book-visit/check-visit-details" method="POST" novalidate>
```

Comment:
Ben's building this page as VB-3481 - just added the minimum here to make it work!